src/build.rs

```diff
@@ -61,10 +61,16 @@ impl Builder {
 
     // Remove any old or duplicate json files.
     fn clean_analysis(&self) {
-        let crate_names = self.crate_names();
+        let crate_names = crate_names()
+            .map(|name| name.replace("-", "_"))
```

Comment:
Good catch!

Reply:
If you want to avoid duplicating the collect, you could move this map to `crate_names` to avoid the new complexity there.

Reply:
Or could combine with the following `map` on line 67

Reply:
You need to collect twice to have somewhere for the `&str`s to point to

Reply:
I think it is worth using `any` rather than `contains` for the sake of cleaner code here and in `crate_names()`, if not for performance.